pconvert_rust/test/blending.py

```diff
@@ -0,0 +1,99 @@
+            os.path.abspath(f"{TEST_ASSETS}shoelace.png"),
+            os.path.abspath(f"{TEST_ASSETS}result_alpha_alpha_Fast_NoFilter.png"),
+        )
+
+        pconvert_rust.blend_images(
+            os.path.abspath(f"{TEST_ASSETS}result_alpha_alpha_Fast_NoFilter.png"),
+            os.path.abspath(f"{TEST_ASSETS}background_alpha.png"),
+            os.path.abspath(f"{TEST_ASSETS}result_alpha_alpha_Fast_NoFilter.png"),
+        )
+
+    def test_blend_images_multithread(self):
+        pconvert_rust.blend_images(
+            os.path.abspath(f"{TEST_ASSETS}sole.png"),
+            os.path.abspath(f"{TEST_ASSETS}back.png"),
+            os.path.abspath(f"{TEST_ASSETS}result_alpha_alpha_Fast_NoFilter.png"),
+            options = {
+                "num_threads": 5
+            }
+        )
+
+        pconvert_rust.blend_images(
+            os.path.abspath(f"{TEST_ASSETS}result_alpha_alpha_Fast_NoFilter.png"),
+            os.path.abspath(f"{TEST_ASSETS}front.png"),
+            os.path.abspath(f"{TEST_ASSETS}result_alpha_alpha_Fast_NoFilter.png"),
+            options = {
+                "num_threads": 5
+            }
+        )
+
+        pconvert_rust.blend_images(
+            os.path.abspath(f"{TEST_ASSETS}result_alpha_alpha_Fast_NoFilter.png"),
+            os.path.abspath(f"{TEST_ASSETS}shoelace.png"),
+            os.path.abspath(f"{TEST_ASSETS}result_alpha_alpha_Fast_NoFilter.png"),
+            options = {
+                "num_threads": 5
+            }
+        )
+
+        pconvert_rust.blend_images(
+            os.path.abspath(f"{TEST_ASSETS}result_alpha_alpha_Fast_NoFilter.png"),
+            os.path.abspath(f"{TEST_ASSETS}background_alpha.png"),
+            os.path.abspath(f"{TEST_ASSETS}result_alpha_alpha_Fast_NoFilter.png"),
+            options = {
+                "num_threads": 5
+            }
+        )
+
+    def test_blend_multiple(self):
+        pconvert_rust.blend_multiple(
+            (
+                os.path.abspath(f"{TEST_ASSETS}sole.png"),
+                os.path.abspath(f"{TEST_ASSETS}back.png"),
+                os.path.abspath(f"{TEST_ASSETS}front.png"),
+                os.path.abspath(f"{TEST_ASSETS}shoelace.png"),
+                os.path.abspath(f"{TEST_ASSETS}background_alpha.png")
+            ),
+            os.path.abspath(f"{TEST_ASSETS}result_alpha_alpha_Fast_NoFilter.png")
+        )
+
+    def test_blend_multiple_multithread(self):
+        pconvert_rust.blend_multiple(
+            (
+                os.path.abspath(f"{TEST_ASSETS}sole.png"),
+                os.path.abspath(f"{TEST_ASSETS}back.png"),
+                os.path.abspath(f"{TEST_ASSETS}front.png"),
+                os.path.abspath(f"{TEST_ASSETS}shoelace.png"),
+                os.path.abspath(f"{TEST_ASSETS}background_alpha.png")
+            ),
+            os.path.abspath(f"{TEST_ASSETS}result_alpha_alpha_Fast_NoFilter.png"),
+            options = {
+                "num_threads": 5
+            }
+        )
```

Comment:
`cleanup .` to get rid of the problems with newlines.

Also, you can create `.cignore` files in folders where you don't want to run it (e.g. `node_modules`).